File: beginners/1-problemset-estruturas-basicas/C++/1010-calculo-simples/calculo_simples.cpp
#include <iostream>
using namespace std;

int main() {
    int a, b, c, d;
    double f, g, total;

    cin >> a >> b >> f >> c >> d >> g;

    total = (b * f) + (d * g);

    printf("VALOR A PAGAR: R$ %.2lf\n", total);

    return 0;
}

File: beginners/1-problemset-estruturas-basicas/Python/1010-calculo-simples/calculo_simples.py
valores = []
pecas = []

for i in range(2):
    cod_peca, num_peca, valor = input().split()
    cod_peca = int(cod_peca)
    num_peca = int(num_peca)
    valor = float(valor)
    pecas.append(num_peca)
    valores.append(valor)

valor_total = (valores[0] * pecas[0]) + (valores[1] * pecas[1])

print(f'VALOR A PAGAR: R$ {valor_total:.2f}')
